tests/GameTests.cpp

```diff
@@ -95,3 +95,97 @@ TEST_F(GameTests, score_should_be_three)
     ASSERT_EQ(3, checkValue);
 }
 
+TEST_F(GameTests, first_ball_should_be_3_and_second_4)
+{
+    //GIVEN
+    Game game;
+    int firstBall, secondBall;
+    //WHEN
+    std::tie(firstBall, secondBall) = game.convertValueToInt("34");
+    //THEN
+    ASSERT_EQ(3, firstBall);
+    ASSERT_EQ(4, secondBall);
+}
+
+TEST_F(GameTests, first_ball_should_be_10_and_second_0)
+{
+    //GIVEN
+    Game game;
+    int firstBall, secondBall;
```

Comment:
Please extract common variables into GameTests class, so that you will not have to create them in every test.